draft-ietf-tls-esni.md

```diff
@@ -1445,9 +1445,13 @@ following capabilities:
 1. The attacker does not know the ECHConfigList used by the server.
 1. The attacker keeps per-connection state only. In particular, it does not
    track endpoints across connections.
-1. ECH and GREASE ECH are designed so that the following features do not vary:
-   the code points of extensions negotiated in the clear; the length of
-   messages; and the values of plaintext alert messages.
+
+Moreover, real ECH and GREASE ECH are designed so that the following features
+do not vary:
```

Comment:
```suggestion
Moreover, real ECH and GREASE ECH are designed so that implementations can ensure the following features
do not vary significantly:
```

Reply:
I don't really know if this is a helpful qualifier because we don't have a definition of "significantly," so I think I'd prefer to leave this off for now.